lib/geordi/commands/dump.rb

```diff
@@ -55,8 +55,8 @@ def dump(target = nil, *_args)
     Interaction.announce "Dumping the database of #{target}#{database_label}"
     dump_path = Geordi::Remote.new(target).dump(options)
 
-    if options.load || options.load_database # … and dump loading
-      loader = DumpLoader.new(dump_path, options[:load_database])
+    if options.load || options.database # … and dump loading
```

Comment:
The addition of `options.database` is not correct: `geordi dump staging -d app` should dump the remote "app" database, but not source it locally.